### DISCUSSION TITLE
Morphine in a headset with an injector is used first.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#13691] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

If morphine is inserted into a headset with an injector.
When you select morphine in the treatment interface, the morphine from the headset is used first.
It is not right. Morphine from the headset should be used last.

### Reproduction steps

1. Place the morphine in the medic uniform in your inventory.
2. Open the treatment interface and use morphine.
3. The morphine from the headset will be the first to be spent

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.2.8.0 (Winter Update hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/issues/12951. This has been fixed, but the fix is unfortunately yet included in the public version of the game.
